docs/docs/platform/compute.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Compute Resources'
+sidebar_position: 1
+image: /img/og/platform/compute-resources.png
+---
+
+Compute resources are the fundamental units that represent the processing power and memory available to your Nhost projects. The primary compute resources are vCPU and RAM. This documentation outlines the key aspects of compute resources in the context of the Nhost Cloud Platform.
+
+
+## Shared vs Dedicated Compute
+
+All free and pro projects use shared vCPUs:
+
+- 2vCPUs / 1GiB for free projects
```

Comment:
should we outline the distribution between services?

Reply:
done